src/modules/show-movies.js

```diff
@@ -0,0 +1,21 @@
+import movieContainer from './project-const.js';
+import createMovies from './display-movies.js';
+
+const showMovies = async () => {
+  movieContainer.innerHTML = '';
+  const movieArr = await createMovies();
+  movieArr.forEach((element) => {
+    movieContainer.innerHTML += `
+     <li class = "movie-${element.id}">
+                <img src="${element.img}" alt="${element.name}" width="200px">
```

Comment:
Kindly avoid using inline CSS and instead add the styling on an external stylesheet for better practice

Reply:
Thank you